coalahtml/_coalahtml/bower.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "coala-html",
   "dependencies": {
-    "angular": "^1.4.0",
-    "angular-route": "^1.4.0",
-    "bootstrap": "^3.3.6",
-    "google-code-prettify": "^1.0.4",
-    "angular-sanitize": "^1.5.0",
-    "bootstrap-material-design": "^0.5.10"
+    "angular": "^1.6.2",
+    "angular-route": "^1.6.2",
+    "bootstrap": "^3.3.7",
+    "google-code-prettify": "^1.0.5",
+    "angular-sanitize": "^1.6.2",
+    "bootstrap-material-design": "^4.0.2"
```

Comment:
this is the only real change in your PR.
Why do we need bootstrap-material-design ^4.0.2?